packages/backend/src/controllers/authentication.ts

```diff
@@ -252,10 +250,14 @@ export const initiateOktaOpenIdLogin: RequestHandler = async (
         const codeVerifier = generators.codeVerifier();
         const codeChallenge = generators.codeChallenge(codeVerifier);
 
+        const extraScopes = lightdashConfig.auth.okta.extraScopes
+            ? ` ${lightdashConfig.auth.okta.extraScopes.join(' ')}`
+            : '';
+
         const authorizationUrl = client.authorizationUrl({
             redirect_uri: redirectUri,
             response_type: 'code',
-            scope: 'openid profile email',
+            scope: 'openid profile email'.concat(extraScopes),
```

Comment:
an additional space is needed after `email` if extra scopes are provided

Reply:
it was there above but now I've changed it back to be closer to the code :+1: